app/assets/javascripts/embed.js

```diff
@@ -26,20 +26,46 @@ require([
      */
     _initApp: function() {
       if (!Backbone.History.started) {
-        Backbone.history.start({pushState: true});
+        Backbone.history.start({
+          pushState: true,
+          root: '/embed'
+        });
       }
     },
 
     _initRouter: function() {
       this.router = new RouterView();
     },
 
-    _initViews: function() {
-      new PantropicalView();
+    _handlebarsPlugins: function() {
+      Handlebars.registerHelper('ifCond', function (v1, operator, v2, options) {
```

Comment:
this helper seems to be defined in multiple files. Could it be placed somewhere where it could just be shared between "everyone" that needs it?